CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -963,10 +963,6 @@ local function CreateSettingsHub()
 
 			local noOpFunc = function() end
 			ContextActionService:BindCoreAction("RbxSettingsHubStopCharacter", noOpFunc, false,
-												 Enum.PlayerActions.CharacterForward, 
```

Comment:
Whats the reasoning behind removing these? I think these stop the character from moving when in the menu?